packages/edit-site/src/components/template-details/index.js

```diff
@@ -56,6 +68,17 @@ export default function TemplateDetails( { template, onClose } ) {
 				) }
 			</div>
 
+			{ isTemplateRevertable( template ) && (
+				<div className="edit-site-template-details__revert">
+					<MenuItem
+						info={ __( 'Restore template to theme default' ) }
+						onClick={ revert }
+					>
+						{ __( 'Clear customizations' ) }
```

Comment:
Not sure if this will be clear, I still find something like `Restore theme defaults` more readable.